lib/rubocop/cop/mixin/comments_help.rb

```diff
@@ -16,6 +16,8 @@ def contains_comments?(node)
       end
 
       def comments_in_range(node)
+        return [].to_enum unless node.source_range
```

Comment:
`to_enum` seems redundant.

```suggestion
        return [] unless node.source_range
```